### PR TITLE
Logging user in, if token in SecureStore

### PR DESCRIPTION
**Pull request by MalteMagnussen**
- [x] Logging user in, if token in secureStore 
- [x] Remove token from user, since its now in SecureStore 
- [x] Test 
- [x] A lot of debugging

**How to test**
1. Run backend, frontend, ngrok, replace ngrok yada yada
2. Log in
3. Close app
4. Reopen app
5. **Expected Behavior:** Should still be logged in

Right now we control the logged in state via a useState. And that is fine. But, since user might have a JWT in the SecureStore, we could make a useEffect, that checks for a token in securestore, and if it is there, then decodes it, and logs the user in? So they keep their logged in, even if they close Expo and the App? :) Could be Cool imo

https://github.com/Hold-Krykke/Security_and_FullstackJavascript_Exam/issues/23

### CODE REVIEW

File: FindYourFriends/screens/LoginScreen.js
Comment (MrHardcode) [inline FindYourFriends/screens/LoginScreen.js]: I agree

**Review by MrHardcode — APPROVED** [review]
Good idea, looks nice

File: FindYourFriends/screens/LoginScreen.js
Comment (MalteMagnussen) [reply FindYourFriends/screens/LoginScreen.js]: Did I get 'em all? 355dd6b

**Comment by MalteMagnussen** [comment]
Works perfectly on my phone.

**Review by Castau — APPROVED** [review]

File: Backend/src/resolverMap.ts
Comment (MalteMagnussen) [inline Backend/src/resolverMap.ts:54]: Authentication if user needs to be logged in to do something. 

Forbiddenerror if user is logged in, but doesnt have good enough permissions to do something

File: Backend/src/server.ts
Comment (MalteMagnussen) [inline Backend/src/server.ts]: 60 seconds for testing purposes. 

Should probably be changed before merge. This is in seconds. For 1 hour you can replace it with a string like "1h".

> Eg: 60, "2 days", "10h", "7d". A numeric value is interpreted as a seconds count. If you use a string be sure you provide the time units (days, hours, etc), otherwise milliseconds unit is used by default ("120" is equal to "120ms").

If int, it is seconds. 

```suggestion
const tokenExpirationInSeconds = 3600;
```

File: FindYourFriends/screens/HomeScreen.js
Comment (MalteMagnussen) [inline FindYourFriends/screens/HomeScreen.js:42]: Otherwise it is using cache, and then when you refetch, you think it hasnt logged you properly out (by deleting token etc), when it actually has, and is just getting from cache, and not backend.

**Comment by MalteMagnussen** [comment]
Fixed the Error Message bug. Was a bad switchcase. Seems ready to merge now, after testing by everyone :)

**Pull request merged**
